### DISCUSSION TITLE
Got Error Refused to Create Blob worker when deploy to the server

### DISCUSSION THREAD

**Discussion by rahXephonz** [vercel/next.js#52086] [Help]
### Summary

i need help, i have deploy my nextjs applications on the NGINX server
but when see the console chrome browser and safari

i got an error 

```json
Refused to create a worker from 'blob:https://harvez.fishlog-staging.net/8eab810d-8074-48b7-9cdf-d4baf370544d' because it violates the following Content Security Policy directive: "default-src * data: 'unsafe-eval' 'unsafe-inline'". Note that 'worker-src' was not explicitly set, so 'default-src' is used as a fallback. Note that '*' matches only URLs with network schemes ('http', 'https', 'ws', 'wss'), or URLs whose scheme matches `self`'s scheme. The scheme 'blob:' must be added explicitly.
```

Where this error coming from? i didn't find any solutions for fixing this.
![image](https://github.com/vercel/next.js/assets/77242429/1c6988c9-eeaa-4aaa-834a-16c764458e47)

Any solutions for this?
Please help

### Additional information

```js
next: 12.1.5
next-pwa: 5.5.4
```

**Comment by icyJoseph** [comment]
I guess there's an actual CSP setting somewhere in your app? https://nextjs.org/docs/app/api-reference/next-config-js/headers#content-security-policy

You probably need to do as instructed here: https://github.com/shadowwalker/next-pwa/issues/218

**Reply by rahXephonz** [reply]
Hello icy,

i confused at setting up my config headers because the error its different from https://github.com/shadowwalker/next-pwa/issues/218
maybe can you help me on this issue?..

Thank you so much.

**Reply by rahXephonz** [reply]
thanks, i configure it.
